src/Pages/Auth/Register/styles.js

```diff
@@ -0,0 +1,30 @@
+import { css } from "emotion"
+
+export const emailInput = css`
+  margin-bottom: 24px !important;
+  color: var(--primary-90);
+  font-size: 0.78rem;
+  line-height: 1.44;
+  text-transform: uppercase;
+  font-weight: 700;
+  letter-spacing: 0.75px;
+  input {
+    border-color: var(--global-success-70);
+    background-color: var(--global-success-10);
+    color: var(--global-success-70);
+  }
+`
+
+export const submitBtn = css`
+  margin-bottom: 24px !important;
+  border-radius: 6px;
+  width: 100%;
+  background-color: var(--secondary-70);
+  color: var(--global-white);
+  margin-top: 48px;
+`
+
+// I was unable to add margin bottom to the labels of each input, so I added margin top on the input itself.
```

Comment:
Please remove comments like this. You can add it as a comment on github. This doesn't really help the future developer.